src/test/java/com/znsio/teswiz/runner/AppPathTest.java

```diff
@@ -0,0 +1,105 @@
+package com.znsio.teswiz.runner;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AppPathTest {
+    private static final String tempFolderPath = System.getProperty("user.dir") + File.separator + "temp";
```

Comment:
add unitTests as a subfolder inside temp

Reply:
done, added "unitTests" as extra repo in directory variable names where it is necessary